drivers/base/bootvid/common.c

```diff
@@ -15,8 +15,61 @@ ULONG VidpScrollRegion[4] =
     SCREEN_HEIGHT - 1
 };
 
+const PALETTE_ENTRY DefaultPalette[BV_MAX_COLORS] =
+{
+    {0x00, 0x00, 0x00}, /* Black */
+    {0x00, 0x00, 0x80}, /* Blue */
+    {0x00, 0x80, 0x00}, /* Green */
+    {0x00, 0x80, 0x80}, /* Cyan */
+    {0x80, 0x00, 0x00}, /* Red */
+    {0x80, 0x00, 0x80}, /* Magenta */
+    {0x80, 0x80, 0x00}, /* Brown */
+    {0xC0, 0xC0, 0xC0}, /* Light Gray */
+    {0x80, 0x80, 0x80}, /* Dark Gray */
+    {0x00, 0x00, 0xFF}, /* Light Blue */
+    {0x00, 0xFF, 0x00}, /* Light Green */
+    {0x00, 0xFF, 0xFF}, /* Light Cyan */
+    {0xFF, 0x00, 0x00}, /* Light Red */
+    {0xFF, 0x00, 0xFF}, /* Light Magenta */
+    {0xFF, 0xFF, 0x00}, /* Yellow */
+    {0xFF, 0xFF, 0xFF}, /* White */
+};
+
+static BOOLEAN ClearRow = FALSE;
+
 /* PRIVATE FUNCTIONS **********************************************************/
 
+VOID
+NTAPI
+InitializePalette(VOID)
+{
+    ULONG i, pal;
+
+    /* Loop all the entries and set their palettes */
```

Comment:
Maybe you should write a comment somewhere saying that the default palette has R and B components exchanged (this would explain the operation done below).

Reply:
Done, placed a comment before the default palette.